package.json

```diff
@@ -19,18 +19,21 @@
     "url": "https://github.com/tj/commander.js.git"
   },
   "scripts": {
-    "lint": "eslint index.js \"tests/**/*.js\"",
+    "lint": "eslint index.js esm.mjs \"tests/**/*.js\"",
     "typescript-lint": "eslint typings/*.ts",
     "test": "jest && npm run test-typings",
+    "test-esm": "node --experimental-modules ./tests/esm-test.mjs",
     "test-typings": "tsc -p tsconfig.json",
     "typescript-checkJS": "tsc --allowJS --checkJS index.js --noEmit",
-    "test-all": "npm run test && npm run lint && npm run typescript-lint && npm run typescript-checkJS"
+    "test-all": "npm run test && npm run lint && npm run typescript-lint && npm run typescript-checkJS && npm run test-esm"
   },
-  "main": "index",
+  "main": "./index.js",
   "files": [
     "index.js",
+    "wrapper.mjs",
```

Comment:
looks like a typo, must be `esm.mjs`

Reply:
Oh, good catch, thanks! I changed the name half way through.

Reply:
(For interest, I thought `wrapper` was a suitable name for a wrapper used with conditional exports and not seen by normal client use. But when I changed to deep import and the name became visible, I wanted something more meaningful.)

Reply:
Fixed in #1443